src/marqo/core/models/marqo_index.py

```diff
@@ -40,6 +40,11 @@ class FieldType(str, Enum):
     ImagePointer = 'image_pointer'
     MultimodalCombination = 'multimodal_combination'
     CustomVector = "custom_vector"
+    MapNumerical = "map_numerical"
+    MapInt = 'map<text, int>'
```

Comment:
Note we need to accept `map<text,int>`, `map<text,   int>` etc (different numbers of whitespace)